app/controllers/volunteers_controller.rb

```diff
@@ -222,8 +222,8 @@ def knight
     end
 
     volunteer = Volunteer.find(params[:volunteer_id])
```

Comment:
So I _think_ the difference between these is that this one makes the volunteer a "super admin" whereas the one in `assignments_controller` makes the volunteer a "region admin".

As far as authorization goes, there's basically 3 roles a user can be: super admin, region admin, or regular volunteer. Volunteers are assigned to a region, and can only do certain things in that region. Region admins are assigned to a region but can do more things than regular volunteers in that region. Super admins can do pretty much everything.

What would you think of renaming this interactor to something like `ToggleSuperAdmin`? Then we could create a separate interactor for `AssignmentsController#knight` named something like `ToggleRegionAdmin`?

I think it's still fine to have it take an ActiveRecord object (`Volunteer`) instead of an `id` though.